scripts/nagios/check-rabbitmq-consumers

```diff
@@ -19,46 +19,49 @@ states = {
     3: "UNKNOWN",
 }
 
-if 'USER' in os.environ and not os.environ['USER'] in ['root', 'rabbitmq']:
+if "USER" in os.environ and not os.environ["USER"] in ["root", "rabbitmq"]:
     print("This script must be run as the root or rabbitmq user")
 
 
 usage = """Usage: check-rabbitmq-consumers --queue=[queue-name] --min-threshold=[min-threshold]"""
 
 parser = argparse.ArgumentParser(usage=usage)
-parser.add_argument('--min-threshold',
-                    dest='min_count',
-                    type=int,
-                    default=1,
-                    action='store')
+parser.add_argument("--min-threshold", dest="min_count", type=int, default=1, action="store")
 
 options = parser.parse_args()
 
 config_file = configparser.RawConfigParser()
 config_file.read("/etc/zulip/zulip.conf")
+
+
 def get_config(section: str, key: str, default_value: str) -> str:
     if config_file.has_option(section, key):
         return config_file.get(section, key)
     return default_value
-TORNADO_PROCESSES = int(get_config('application_server', 'tornado_processes', '1'))
 
-output = subprocess.check_output(['/usr/sbin/rabbitmqctl', 'list_consumers'],
-                                 universal_newlines=True)
+
+TORNADO_PROCESSES = int(get_config("application_server", "tornado_processes", "1"))
+
+output = subprocess.check_output(
+    ["/usr/sbin/rabbitmqctl", "list_consumers"], universal_newlines=True,
+)
 
 consumers: Dict[str, int] = defaultdict(int)
 
 sys.path.append(os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(__file__)))))
-queues = set(normal_queues).union({
-    # These queues may not be present if settings.TORNADO_PROCESSES > 1
-    'notify_tornado',
-})
+queues = set(normal_queues).union(
+    {
+        # These queues may not be present if settings.TORNADO_PROCESSES > 1
+        "notify_tornado",
+    },
+)
```

Comment:
This seems like it'd be happier with a `tornado_queues` variable that gets unioned in.